frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -374,6 +379,18 @@ function ReaderFooter:init()
     end
     -- require("logger").dbg(self.mode_nb, self.mode_index)
 
+    -- Container settings
+    if not self.settings.container_height then
+        self.settings.container_height = DMINIBAR_CONTAINER_HEIGHT
+    else
+        self.height = Screen:scaleBySize(self.settings.container_height)
+    end
+    if not self.settings.container_bottom_padding then
+        self.settings.container_bottom_padding = 1 -- unscaled_size_check: ignore
+    else
+        self.bottom_padding = Screen:scaleBySize(self.settings.container_bottom_padding)
+    end
```

Comment:
The `if else` logic seems a bit odd (on first exec, if feels like you forgot to set self.height and self.bottom_padding, but ok, you're using the initial values - but you set them in settings - on next exec, you'll use them from settings and are computing them). It works, but it's a bit unexplicite (you have to look around to be sure it's not a bug).
I would just drop the else and do the computation in all cases, for clarity :)

Reply:
Fair enough ;).